Makefile

```diff
@@ -56,6 +57,9 @@ ifeq ($(LUAROCKS_VER),luarocks 3.)
 else
 	luarocks install rockspec/apisix-master-0.rockspec --tree=deps --only-deps --local
 endif
+ifneq ($(LUA_VER),Lua 5.1)
+	sed -i '' -e "s|#!/usr/bin/env lua|#!/usr/bin/env $(LUAJIT_DIR)/bin/luajit|g" ./bin/apisix
```

Comment:
this is a hack way. it should not be enough.

here is another way, we can make a try:

```
#!/bin/sh

# find the openresty, eg: which openresty
...
# find the luajit of openresty
...
# run the CLI of APISIX
$luajit -e '...'
```

Reply:
@membphis
I don't understand very clearly. 
Do you mean to create a new shell script to judge whether to use lua5.1 or luajit when running the CLI of APISIX?
It could run the CLI of APISIX selectively: `lua ./bin/apisix ` or `luajit -e ./bin/apisix`?

Reply:
we can use a shell script to write the CLI, it'll to call `luajit ./bin/apisix`